pkg/search/proxy/store/multi_cluster_cache.go

```diff
@@ -52,10 +52,10 @@ type Store interface {
 
 // MultiClusterCache caches resource from multi member clusters
 type MultiClusterCache struct {
-	lock            sync.RWMutex
-	cache           map[string]*clusterCache
-	cachedResources map[schema.GroupVersionResource]struct{}
-	restMapper      meta.RESTMapper
+	lock             sync.RWMutex
+	cache            map[string]*clusterCache
+	matchedResources map[schema.GroupVersionResource]struct{}
```

Comment:
How about rename `matched` to `registried`?

Reply:
OK，registered